Tensile/LibraryLogic.py

```diff
@@ -423,6 +423,12 @@ def addFromCSV(self, dataFileName, numSolutions, solutionMap):
     # notice that for OperationType != GEMM, the numIndices = 0
     totalSizeIdx = problemSizeStartIdx + self.numIndices
 
+    if globalParameters["CSVExportWinner"]:
+      # the column of the two are fixed (GFlops, SizeI/J/K/L, LDD/C/A/B, TotalFlops, WinnerGFlops, WinnerTimeUs, WinnerIdx, WinnerName) 
+      # the order are implemented in ResultFileReporter.cpp (NewClient) and Client.h (OldClient)
+      columnOfWinnerGFlops = 10
+      columnOfWinnerIdx = 12
+
     # iterate over rows
     rowIdx = 0
     for row in csvFile:
```

Comment:
this will analyze all csvFile in same folder,
need to separate csvFile with CSVExportWinner:True and csvFile with CSVExportWinner:False.
or it will use CSVExportWinner:True format to analyze CSVExportWinner:False csv, and get indexError.

Reply:
Yes, you're right, thanks for reminding. I've fixed this and rebase again